public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
🤔 Not sure if this was intentionally omitted before, but it was generated by create-react-app